build/gen-lint-exclude.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+excluded=$(find . -name "*.go" | grep -v vendor | xargs grep autogenerated | cut -d ':' -f 1 | uniq | sed 's#\./##g')
```

Comment:
This should probably go in [build/build-image](https://github.com/GoogleCloudPlatform/agones/tree/master/build/build-image) with the rest of the gen scripts.

Which will mean it will be [automatically copied](https://github.com/GoogleCloudPlatform/agones/blob/master/build/build-image/Dockerfile#L68) into the build Docker image so it will be easier to call.

May also want to also bump the [ForceUpdate](https://github.com/GoogleCloudPlatform/agones/blob/master/build/build-image/Dockerfile#L15) value to force the rebuild.